deployment/hasura/metadata/databases/tables/sequencing/parcel.yaml

```diff
@@ -0,0 +1,60 @@
+table:
+  name: parcel
+  schema: sequencing
+configuration:
+  custom_name: "parcel"
+object_relationships:
+  - name: channel_dictionary
+    using:
+      foreign_key_constraint_on: channel_dictionary_id
+  - name: command_dictionary
+    using:
+      foreign_key_constraint_on: command_dictionary_id
+  - name: sequence_adaptation
+    using:
+      foreign_key_constraint_on: sequence_adaptation_id
+array_relationships:
+  - name: parameter_dictionary
```

Comment:
Array relationships should have "plural" names, i.e `parameter_dictionaries`.

Reply:
Thank you for caught that, I made it plural.